client/stylesheets/_modal.scss

```diff
@@ -247,7 +247,7 @@
 	user-select: none;
 }
 
-.modOfferRequestPaymentWrap ul li input.offerRequestPaymentRadio:empty ~ label:before {
+.radioWrap ul li input.radioWrapElement:empty ~ label:before {
```

Comment:
Merge rule `.radioWrap ul li input.radioWrapElement:empty ~ label:before` with rule on line 215<br>Begin pseudo elements with double colons: `::`<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector `radioWrap` should be written in lowercase with hyphens<br>Selector `radioWrapElement` should be written in lowercase with hyphens